src/supautils.c

```diff
@@ -901,6 +1016,36 @@ privileged_extensions_check_hook(char **newval, void **extra, GucSource source)
 	return true;
 }
 
+static bool
+privileged_role_allow_policies_on_tables_check_hook(char **newval, void **extra, GucSource source)
+{
+	List *table_list;
+	ListCell *table;
+	char *val = *newval;
+
+	if (val != NULL)
+	{
+		if (!SplitIdentifierString(pstrdup(val), ',', &table_list))
+			EREPORT_INVALID_PARAMETER("supautils.privileged_role_allow_policies_on_tables");
```

Comment:
Return false here?

Reply:
This throws an error (no idea how that works in C), so no need for return